docs/conf.py

```diff
@@ -1,33 +1,37 @@
+import datetime as dt
 import os
+import re
 import sys
-import datetime as dt
+from pathlib import Path
+
+MATCH_VERSION_LINE = re.compile(r"version = \W((\d+\.\d+)\.\d.*?)\W").fullmatch
 
+pyproject = Path(__file__).parent.parent / "pyproject.toml"
```

Comment:
We should consider using [tomllib](https://docs.python.org/3/library/tomllib.html) for this instead. Was only added recently in python 3.11 tho.